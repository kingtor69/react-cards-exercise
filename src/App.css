.App {
	text-align: center;
}

button {
	background-color: blue;
	color: white;
	border-radius: 0.3em;
	margin: 0.1em;
}

.big-btn {
	font-size: 2em;
}

.table {
	position: relative;
}

.Card {
	position: absolute;
}
