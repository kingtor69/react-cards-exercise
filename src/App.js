import React from 'react';
import './App.css';
import {deckBaseUrl, deckNewEnd, deckDrawEnd} from './helpers';
import Cards from './Cards';

function App() {
  return (
    <div className="App">
      <Cards 
        deckBaseUrl={deckBaseUrl}
        deckNewEnd={deckNewEnd}
        deckDrawEnd={deckDrawEnd}
      />
    </div>
  );
}

export default App;
