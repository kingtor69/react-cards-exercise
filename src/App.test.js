import { render } from '@testing-library/react';
import App from './App';

it('renders without crashing', () => {
  render(<App />);
});

it('renders snapshot', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});