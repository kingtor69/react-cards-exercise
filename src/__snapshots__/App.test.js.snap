// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="Cards"
    >
      <h2>
        Let's throw some cards around, eh?
      </h2>
      <button
        class="big-btn"
      >
        shuffle 1 deck
      </button>
      <br />
      <button>
        more decks
      </button>
      <button>
        fewer decks
      </button>
    </div>
  </div>
</DocumentFragment>
`;
